Projects/Server/MultiGameRunner.h

```diff
@@ -0,0 +1,28 @@
+#ifndef MULTIGAMERUNNER_H
+#define MULTIGAMERUNNER_H
+
+#include "State.h"
+#include "GameHandler.h"
+
+#include <map>
+
+class MultiGameRunner 
+{
+public:
+	MultiGameRunner(const AbstractPlayer::Ptr& player0, const AbstractPlayer::Ptr& player1, const size_t& gameSample);
```

Comment:
```suggestion
	MultiGameRunner(const AbstractPlayer::Ptr& player0, const AbstractPlayer::Ptr& player1, const size_t& totalGamesToBePlayed);
```
To stay in line with implementation